res/Sectors/M1105/M1105.xml

```diff
@@ -3210,6 +3254,72 @@ Left:   0041 0040 0039 0038 0037 0036 0035 0034 0033 0032 0031 0030 0029 0028 00
     <Name Lang="hv" Publisher="HIWG">Ultret</Name>
   </Sector>
 
+  <!-- Names of the Rim Route -->
+  <Sector>
+    <Y>9</Y>
+    <X>8</X>
+    <Name Publisher="HIWG">Binaurie</Name>
```

Comment:
Is HIWG really appropriate here, or are these taken from Bari Stafford?

Reply:
I did it like this as this is now the other Rim Route inspired Sectors that have just names are marked. I guess these are certainly Bari Stafford sourced, so proper credit is due for them.

Should this be ...

<Name Source="The Core Route Project" Author="Bari Z. Stafford Sr." Ref="http://web.archive.org/web/20030406042613/http://www.geocities.com/traveller_core_route/proj004/proj004.html">Binaurie</Name>

Or just the author?

<Name Author="Bari Z. Stafford Sr.">Binaurie</Name>

Reply:
Full details could be helpful for future researchers.

Reply:
OK, I have corrected these (and the mistake of locations for the last 2 sectors)